doc/integrator/preparativework.rst

```diff
@@ -0,0 +1,75 @@
+.. _integrator_preparativework:
+
+Creating a SSH Key with Windows
+==================================
+
+1. Download PuTTYgen:
+
+   a) Windows 95, 98, ME, NT, 2000 and XP on Intel x86: http://the.earth.li/~sgtatham/putty/latest/x86/puttygen.exe
```

Comment:
What about more recent versions of Windows?
On the download page I don't really see links for 64-bits versions...